### ISSUE TITLE
Datagrid customsource Error while using DevExtreme.AspNet.Data

### ISSUE THREAD

**Issue by salemu01** [DevExpress/DevExtreme#17548]
### Steps to Reproduce:

1. With an angular application, and devextreme@21.1 beta
2. Use a datagrid, with custom datasource with DevExtreme.AspNet.Data as such:

________________________________Typescript____________________________
```
import { createStore } from 'devextreme-aspnet-data-nojquery';
import DataSource from 'devextreme/data/data_source';
.....
this.dataSource = new DataSource({
      store: createStore({
       ...
      }),
...
```
________________________________HTML_________________________________
```
  <dx-data-grid  [dataSource]="dataSource" ......./>
```

### Results You Received:
Code working on 20.X versions, but on the beta version, we have the following javascript error:

```
core.js:6157 ERROR TypeError: CustomStore is not a constructor 
at createStore (index.js:398) 
at GridWidgetComponent.CreateDataSource (grid-widget.component.ts:148) 
at SafeSubscriber._next (grid-widget.component.ts:90) 
at SafeSubscriber.__tryOrUnsub (Subscriber.js:183) 
at SafeSubscriber.next (Subscriber.js:122) 
at Subscriber._next (Subscriber.js:72) 
at Subscriber.next (Subscriber.js:49) 
at ReplaySubject._subscribe (ReplaySubject.js:61) 
at ReplaySubject._trySubscribe (Observable.js:42) 
at ReplaySubject._trySubscribe (Subject.js:81)
```

### Results You Expected:

Instance of CustomStore created without issue. There seems to be a problem with package DevExtreme.AspNet.Data (devextreme-aspnet-data-nojquery) compatibility with the changes made in this beta version 21.1.

### Environment Details:

Angular 11
Devextreme 21.1beta
devextreme-aspnet-data-nojquery 2.8.2

**Comment by AlekseyMartynov** [comment]
Would you please check a fixed package?
https://www.npmjs.com/package/devextreme-aspnet-data-nojquery/v/2.8.4

**Comment by salemu01** [comment]
Hello,
Thank you for you fast return. I will test this package and give you feedback.

________________________________

**Comment by salemu01** [comment]
The problem has been fixed in this new 2.8.4 version. 

Thank you AlekseyMartynov, and the devexpress team.

**Comment by heyimnowi** [comment]
Hello, is not working for you. Any luck you guys?
```

import CustomStore from 'devextreme/data/custom_store';

  makeAsyncDataSource(jsonFile) {
    return new CustomStore({
      loadMode: 'raw',
      key: 'ID',
      load: async () => of(jsonFile)
    });
  }
```

ERROR TypeError: CustomStore is not a constructor
